index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Miftah's Protfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500&family=Roboto&display=swap"/>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"/>
+</head>
+<body>
+    <header>
+        <a href="#" class="mylogo">Miftah</a>
+        <nav class="mob-version">
+           <i class="fa fa-bars">
```

Comment:
- [ ] Please make sure to use the same colors (or the most similar ones) as well as the fonts within your design to try to match the Figma template. Here, your hamburger menu is black while in the original design it's white. Kindly change it.

Reply:
Changed the color, Sorry that I was not attentive to it.